packages/vx-shape/src/shapes/AreaClosed.js

```diff
@@ -13,7 +13,7 @@ export default function AreaClosed({
   strokeDasharray,
   strokeWidth = 2,
   stroke = 'black',
-  fill = rgba(0,0,0,0.3),
+  fill = "rgba(0,0,0,0.3)",
```

Comment:
beautifier as in an IDE plugin of sorts? I'm not totally sure about that one but harry might know. 

I think adding linting to the root  of the `vx` repo and running it for travis could be a good call for catching things like this (and shouldn't require adding it to every package). the number of errors flagged on the initial pass can be a little rough though and it will flag things like `propTypes` + `defaultProps` not being defined for components but could add them to the ignore list to start. Happy to give this a shot as well / recently set it up for the @data-ui repo.

I'd imagine @harry has probably given this all some thought already, though?